Common/cpp/SharedItems/ShareableValue.cpp

```diff
@@ -268,16 +268,16 @@ jsi::Value ShareableValue::toJSValue(jsi::Runtime &rt) {
             size_t count
             ) -> jsi::Value {
 
-          jsi::Value jsThis = rt.global().getProperty(rt, "jsThis");
-          std::string workletLocation = jsThis.asObject(rt).getProperty(rt, "__location").toString(rt).utf8(rt);
+         // jsi::Value jsThis = rt.global().getProperty(rt, "jsThis");
+         // std::string workletLocation = jsThis.asObject(rt).getProperty(rt, "__location").toString(rt).utf8(rt);
```

Comment:
Was it removed temporarily or permanently?

Reply:
I don't remember it perfectly, but there was a case in which host function was called and jsThis hadn't been
 set.